src/legacy/core_plugins/dashboard_embeddable_container/public/np_ready/public/tests/dashboard_container.test.tsx

```diff
@@ -32,14 +32,14 @@ import { getSampleDashboardInput } from '../lib/test_helpers';
 import {
   CONTACT_CARD_EMBEDDABLE,
   ContactCardEmbeddableFactory,
-} from '../../../../../embeddable_api/public/np_ready/public/lib/test_samples/embeddables/contact_card/contact_card_embeddable_factory';
+} from '../../../../../embeddable_api/public/np_ready/public/lib/test_samples';
```

Comment:
can these not be imported from new platform?  Why keep in legacy?

Reply:
Next PR will move `dashboard_embeddable_container` to NP. I had PR before this (which I closed) where I updated these, but it just introduced unnecessary problems with tests and I will need to change these paths again in next PR (so double work).